src/applications/rated-disabilities/components/TotalRatedDisabilities.jsx

```diff
@@ -40,3 +40,21 @@ TotalRatedDisabilities.propTypes = {
 };
 
 export default TotalRatedDisabilities;
+
+// content = (
```

Comment:
Need to determine where this needs to go. Most likely this is the content that should appear when there is a server error on the API call